change/@ni-nimble-components-6aa0f191-427b-4745-9425-f602170a1266.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Add spec for nimble-card, no code changes yet",
+  "packageName": "@ni/nimble-components",
+  "email": "kelsey.johnson@ni.com",
```

Comment:
Most of us don't expose our NI email address in the change files. If you don't want to have your address here, you can follow the [instructions in the CONTRIBUTING doc](https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#beachball-change-file) to configure github to keep your email address private.

Reply:
Fixed. Thanks for the tip!